cnf-tests/testsuites/e2esuite/sctp/sctp.go

```diff
@@ -62,6 +64,10 @@ func init() {
 	if os.Getenv("SCTPTEST_HAS_NON_CNF_WORKERS") == "false" {
 		hasNonCnfWorkers = false
 	}
+
+	var err error
+	isSingleNode, err = utilNodes.IsSingleNode()
```

Comment:
Same as other reviews, please move this inside a test step.

Reply:
Done